src/status_im/ui2/screens/chat/composer/view.cljs

```diff
@@ -160,6 +162,12 @@
                                       (reanimated/set-shared-value bg-opacity (reanimated/with-timing 0)))
                                     (reanimated/set-shared-value translate-y (reanimated/with-timing (- y)))
                                     (reanimated/set-shared-value shared-height (reanimated/with-timing (min y max-height)))))
+              (quo.react/effect! #(when (and (not edit) (= (:state @context) :max))
```

Comment:
this looks weird, why is it separate effect, i see that we set same properties twice, but with different values , im not sure how it will work

Reply:
ok too late

Reply:
but still we need to address that